.github/workflows/virtualbox_image.yml

```diff
@@ -0,0 +1,100 @@
+name: Create VirtualBox disk image from Dockerfile
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  release:
+    types:
+      - published
+  push:
+    paths:
+      - 'docker/*'
+      - '.github/workflows/virtualbox_image.yml'
+
+env:
+  VM_PASSWORD: temppwd
+  DOCKER_IMAGE_BASENAME: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3
+
+
+jobs:
+  pyne_image_build:
+    runs-on: ubuntu-latest
+    outputs:
+      image_tag: ${{ steps.output_tag.outputs.image_tag }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # before merging, change tag-latest-on-default to true
+      - name: Build PyNE docker image
+        uses: firehed/multistage-docker-build-action@v1
+        id: build_pyne
+        with:
+          repository: ${{ env.DOCKER_IMAGE_BASENAME }}
+          stages: openmc
+          server-stage: pyne
+          quiet: false
+          tag-latest-on-default: false
+          dockerfile: docker/ubuntu_22.04-dev.dockerfile
+
+  pushing_test_stable_img:
+    needs: [pyne_image_build]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push Image to stable img
+        uses: akhilerm/tag-push-action@v2.1.0
+        with:
+          src: ${{ env.DOCKER_IMAGE_BASENAME }}/pyne:latest
+          dst: ${{ env.DOCKER_IMAGE_BASENAME }}/pyne:stable
```

Comment:
Since this is running on release, can we get the release name and tag with that as well?

Reply:
According to the [list of default variables](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables), we can use `github.ref` to get the release name if the workflow is triggered by a `release` event. I would like to test this on the `learn-maintenance` repo, but I think that would require me to open a PR with a workflow that runs on `release`, then merge my own PR and create a new release, which I don't know if it's the best practice.

Reply:
It probably doesn't matter so much over there, but I can merge the PR and then let you play with the release

Reply:
(once again too brief in my response) If you want to draft a PR on `learn-maintenace` , I can review/merge it and then we can see how it works - let me know

Reply:
Just opened [a PR](https://github.com/cnerg/learn-maintenance/pull/28) in `learn-maintenance` to add a workflow that will run on `release` and echo the contents of `github.ref`. Hopefully this can be merged soon for testing!